server/tso/allocator_manager.go

```diff
@@ -451,7 +472,9 @@ func (am *AllocatorManager) ClusterDCLocationChecker() {
 			continue
 		}
 		am.mu.clusterDCLocations[dcLocation] = append(am.mu.clusterDCLocations[dcLocation], serverID)
+		am.mu.sortedDCLocations = append(am.mu.sortedDCLocations, dcLocation)
```

Comment:
Will this cause tso duplication when the configuration changes?

Reply:
Added a check. PTAL.